keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/EventController.kt

```diff
@@ -16,7 +16,7 @@ import org.springframework.web.bind.annotation.RestController
 @RestController
 @RequestMapping(path = ["/resources/events"])
 class EventController(
-  private val resourceHistoryService: ResourceHistoryService
+  private val resourceRepository: ResourceRepository
```

Comment:
wasn't the whole point of the service to encapsulate the repository logic? :)

Reply:
Yes. 😞 But... I think the service still has value in that it's encapsulating the logic around determining resource status based on events (which might warrant changing the name of the class, BTW), whereas here we really just needed to retrieve the list of events directly from the data source _without_ any additional processing, and IMHO it's best to avoid additional indirection levels just to proxy calls where we can, because it makes troubleshooting much easier.

Reply:
ok, make sense!